### PR TITLE
GitHub CI

### PR DESCRIPTION
**Pull request by hannesa2**
I tried to build with github action, but because of cascading git submodules it's not that easy, and if one of them doesn't merge a followup pull request, all doesn't work.
Additionally I'm a beginner with XCode, that's why **any help is welcome.**

Related follow up pull requests are
https://github.com/gitx/MGScopeBar/pull/1
https://github.com/hannesa2/objective-git/pull/1 (With Github Action CI)

### CODE REVIEW

**Comment by hannesa2** [comment]
I made for submodules this as well 
https://github.com/gitx/objective-git/pull/3
https://github.com/gitx/MGScopeBar/pull/2

**Comment by hannesa2** [comment]
Travis is green, but it fails when you look in console !
image

I will bring this PR to this repo, to see immediate if Github action fails too

**Comment by jspiro** [comment]
After we do this, does it make sense to continue with Travis?

File: .github/workflows/BuildPR.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: PullRequest
+
+on: [pull_request]
+
+jobs:
```
Comment (jspiro) [inline .github/workflows/BuildPR.yml:5]: Is anything from #198 useful here? Once this works, that can be closed.

Comment (hannesa2) [reply .github/workflows/BuildPR.yml:5]: I see no point to run with python there
`run: python3 Scripts/build.py release`

I will close it

**Comment by hannesa2** [comment]
> After we do this, does it make sense to continue with Travis?

No, Travis make s no sense anymore. But I will delete it after this merge to see how more faster Github actions are

**Comment by hannesa2** [comment]
I close this, because I created it in this repo https://github.com/gitx/gitx/pull/208  to have an instant run

**Pull request closed without merging**